File: openapi/code_samples/python/api_v1_passwordless_email_send/post.py
response = scalekit_client.passwordless.send_passwordless_email(
    email="john.doe@example.com",
    template="SIGNIN",  # or "SIGNUP", "UNSPECIFIED"
    expires_in=100,
    magiclink_auth_uri="https://yourapp.com/auth/passwordless/callback",
    template_variables={
        "employeeID": "EMP523",
        "teamName": "Alpha Team",
        "supportEmail": "support@yourcompany.com",
    },
)

# with_call returns (response, call)
auth_request_id = response[0].auth_request_id

File: openapi/code_samples/javascript/api_v1_passwordless_email_send/post.js
const response = await scalekit.passwordless.sendPasswordlessEmail('john.doe@example.com', {
  template: 'SIGNIN',
  expiresIn: 100,
  magiclinkAuthUri: 'https://yourapp.com/auth/passwordless/callback',
  templateVariables: {
    employeeID: 'EMP523',
    teamName: 'Alpha Team',
    supportEmail: 'support@yourcompany.com',
  },
})
